mods/persistence/modules/world_save/_defines/_serializer.dm

```diff
@@ -0,0 +1,42 @@
+/*
+	Serialized type names
+*/ 
+#define SERIALIZER_TYPE_NULL		"NULL"
```

Comment:
The indentation on all the lines that follow are inconsistent.

Reply:
That's some weirdness with github. Not sure why it does that. I need to insert a space after everything so tabs look like they align on github.